tests/e2e/mixedos/Vagrantfile

```diff
@@ -98,13 +98,16 @@ def provision(vm, role, role_num, node_num)
     # end
     # If using libvirt, use virt-viewer for GUI after bring up the node
     vm.provision :rke2, run: 'once' do |rke2|
-      rke2.env = [install_type]
+      #rke2.env = [install_type]
+      rke2.env = %W[Commit=c351c1dce61b6952265aae8e78e7fe2ed452a343]
```

Comment:
Is this meant to be merged?

Reply:
Crap! This is how I was testing this PR. This is wrong